src/js/edu-benefits/pages/schoolSelection.js

```diff
@@ -6,15 +6,16 @@ import { uiSchema as uiSchemaDate } from '../../common/schemaform/definitions/da
 export default function createSchoolSelectionPage(
   schema,
   fields = ['educationProgram', 'educationObjective', 'nonVaAssistance'],
-  programRequired = false) {
+  programRequired = false,
```

Comment:
Maybe there should be a 1:1 correspondence between these parameters and the fields they affect (see below).